packages/build-info/src/build-systems/nrwl.ts

```diff
@@ -34,6 +57,23 @@ export class Nx extends BaseBuildTool {
     return []
   }
 
+  /** Retrieve the dist directory of a package */
+  async getDist(packagePath: string): Promise<string | null> {
+    if (!this.isIntegrated) {
+      return null
+    }
+
+    const framework = this.project.frameworks.get(packagePath)?.[0]
+
+    if (framework) {
+      const dist = framework.staticAssetsDirectory || framework.build.directory
+      // TODO: make this smarter in the future by parsing the project.json
```

Comment:
Should we cut an issue for this?

Reply:
Done https://github.com/netlify/build/issues/4994